src/core/exprfilter.cpp

```diff
@@ -3327,7 +3330,7 @@ static void VS_CC exprCreate(const VSMap *in, VSMap *out, void *userData, VSCore
                             compiler->addInstruction(op);
                         }
 
-                        d->proc[i] = compiler->getCode();
+                        std::tie(d->proc[i], d->procSize[i]) = compiler->getCode();
```

Comment:
<tuple>

Reply:
what do you mean? could you please elaborate?

Reply:
@AkarinVS `std::tie()` is defined in `<tuple>` which is not included in this file, so I'd guess @sekrit-twc means you should add `#include <tuple>`. Don't take my word for it though.

Reply:
I see. All the checks have passed, so I never suspected a missing include. Done. PTAL.